### DISCUSSION TITLE
Integrating multiple multiomic datasets

### DISCUSSION THREAD

**Discussion by cnwokolo2** [stuart-lab/signac#1061] [Q&A] [answered]
Hi, thank you for providing many helpful vignettes. 
I am trying to integrate four different single-cell multi-omic (RNA & ATAC) datasets. Could you provide or direct me to a vignette that covers this. I have tried combining the tutorial for scATAC integration and the Joint RNA and ATAC seq integration but I keep running out of vector memory.

**Comment by timoast** [comment, marked as answer]
Please see this discussion topic: https://github.com/timoast/signac/discussions/1012
